hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/NSSummaryEndpoint.java

```diff
@@ -724,6 +868,135 @@ private long getTotalSize(long objectId) throws IOException {
     return result;
   }
 
+  private List<OmKeyInfo> listKeysUnderVolume(String volumeName)
+          throws IOException {
+    List<OmKeyInfo> result = new ArrayList<>();
+    if (Strings.isBlank(volumeName)) {
+      return result;
+    }
+
+    Table keyTable = omMetadataManager.getFileTable();
+
+    TableIterator<String, ? extends Table.KeyValue<String, OmKeyInfo>>
+            iterator = keyTable.iterator();
+
+    while (iterator.hasNext()) {
+      Table.KeyValue<String, OmKeyInfo> kv = iterator.next();
```

Comment:
Can we use the iterator.seek() API that let's us jump to the first key with that the volume name in prefix here?